awx/api/templates/swagger/config.yml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
This is a .yaml file that the docs team can use to control the ordering of the categories in the generated Swagger doc.

Reply:
@tvo318 @docschick ^